### PR TITLE
Add txes from fns

### PR DESCRIPTION
**Pull request by daemianmack**
Howdy sailor. This PR will allow transactions emitted by Clojure code.

Conformity covers a lot of use cases but a pain point I keep running into in my current projects is the need to A) alter lots of existing data that B) isn't reachable via a pull ref, leaving no straight-forward handle on alteration.

Does this seem like a useful addition?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,9 +68,22 @@ You can see this more directly illustrated in a console…
...
+
+Instead of using the `:txes` key to point to an inline transaction, you can also use a `:txes-fn` key pointing to a symbol reference to a function, as follows...
+
+```clojure
+;; resources/something.edn
+{:my-project/something-else-schema
+  {:txes-fn 'my-project.migrations.txes/insert-seeds}}
+```
+
+This function will be passed the Datomic connection and should return transaction data in the format described above, allowing transactions to be driven by full-fledged inspection of the database.
```
Comment (rkneufeld) [inline README.md]: Any chance of showing the params signature for clarity?

File: src/io/rkn/conformity.clj [hunk 2]
```diff
@@ -137,22 +137,39 @@
            (throw (ex-info reason data t))))))
    acc (map-indexed vector txes)))
 
+(defn eval-txes-fn
+  [conn txes-fn]
+  (try (require (symbol (namespace txes-fn)))
+       {:txes ((resolve txes-fn) conn)}
+       (catch Throwable t
+         {:exc (str "Exception evaluating " txes-fn ": " t)})))
```
Comment (rkneufeld) [inline src/io/rkn/conformity.clj]: Maybe `:ex`? I found exc jarring when I saw it first below without context.

File: src/io/rkn/conformity.clj [hunk 3]
```diff
@@ -137,22 +137,39 @@
...
+
+(defn get-norm
+  [conn norm-map norm-name]
```
Comment (rkneufeld) [inline src/io/rkn/conformity.clj]: You would be by hero if you added a docstring.

**Comment by daemianmack** [comment]
Should be all addressed. Thoughts on version number?

**Comment by rkneufeld** [comment]
I'd say bump the minor. I can release today around 11 Central.

**Comment by rkneufeld** [comment]
I really like the example. Sets a tone for how to use appropriately.

**Comment by rkneufeld** [comment]
Released <https://clojars.org/io.rkn/conformity/versions/0.5.0>

**Pull request merged**